__tests__/app.test.js

```diff
@@ -267,59 +356,61 @@ describe("/api/reviews/:review_id/comments", () => {
-        body: "This is a test comment.",
-      };
-      return request(app)
-        .post("/api/reviews/2/comments")
-        .send(invalidComment)
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toEqual("invalid username");
-        });
-    });
-    test('invalid post body results in status 400 - msg "bad request"', () => {
-      const invalidComment = {
-        username: "dav3rid",
-        body: null,
-      };
-      return request(app)
-        .post("/api/reviews/2/comments")
-        .send(invalidComment)
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toEqual("bad request");
-        });
-    });
-    test('empty post object results in status 400 - msg "bad request"', () => {
-      const invalidComment = {};
-      return request(app)
-        .post("/api/reviews/2/comments")
-        .send(invalidComment)
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toEqual("bad request");
-        });
+    describe("POST errors", () => {
+      test("non-existing review id results in status 404 - msg 'review not found", () => {
+        return request(app)
+          .post("/api/reviews/1000/comments")
+          .send(comment)
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toEqual("review not found");
+          });
+      });
+      test("non-numeric review id results in status 400 - msg 'bad request", () => {
+        return request(app)
+          .post("/api/reviews/review1/comments")
+          .send(comment)
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toEqual("bad request");
+          });
+      });
+      test('invalid post username results in status 400 - msg "invalid username"', () => {
+        const invalidComment = {
+          username: "invalidUsername",
+          body: "This is a test comment.",
+        };
+        return request(app)
+          .post("/api/reviews/2/comments")
+          .send(invalidComment)
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toEqual("invalid username");
+          });
+      });
+      test('invalid post body results in status 400 - msg "bad request"', () => {
+        const invalidComment = {
+          username: "dav3rid",
+          body: null,
+        };
+        return request(app)
+          .post("/api/reviews/2/comments")
+          .send(invalidComment)
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toEqual("bad request");
+          });
+      });
+      test('empty post object results in status 400 - msg "bad request"', () => {
+        const invalidComment = {};
+        return request(app)
+          .post("/api/reviews/2/comments")
+          .send(invalidComment)
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toEqual("bad request");
+          });
+      });
     });
```

Comment:
You're destructuring a key from the response body in some of these tests, but not others.  I'd suggest that, in a quiet moment, you make a definitive decision about how you want to deal with your responses, and be consistent throughout the file